### ISSUE TITLE
Food Changes In city / Food Business Propblem

### ISSUE THREAD

**Issue by ayushpatel1007** [InZidiuZ/op-framework-issue-tracker#2902]
### Assurance

I have read the `README.md`

### Summary

There are lots of options from where people can buy/get food. They can grill the meat from hunting , but from vending machines all over the city , also buy very cheap food items from Tuco van , grocery store , fruit shop etc 

This is damaging the food businesses in city. 
I am not owner of any company but I still think this is an issue. 

### Reason

People can buy food from business like arcade , burgershot , bean machine , Bahamas , SOL-Cafe , sushi , Irish pub etc. tere are tons of business to but food due to which there is already so much competition within businesses. Removing vending machines and grocery store food will increase business growth and also increase rp for food business. 

People currently don’t even bother to buy food from this business. Just hunt and eat meat or buy tons of food from Tuco van, or eat fruits and make milk shake using fruit.

Removing other options of getting food expect to get them from business make increase rp for all food business.

**Comment by coalaura** [comment]
no. Grocery store food will always be there for times where other business are not open. Grocery store food is very expensive and barely fills you up so there is no competition in that direction. I will nerf vending machines a bit. The taco truck has a very limited quantity and i'll look at the fruit stand again
